Jenkinsfile

```diff
@@ -53,7 +51,13 @@ pipeline{
       }
       agent {label 'devExpLinuxPool'}
       steps{
-        runtests('Latest','11','JAVA11_HOME_DIR')
+        sh label:'mlsetup', script: '''#!/bin/bash
+            sudo /usr/local/sbin/mladmin remove
```

Comment:
What do these two lines do? I'm thinking that since they're repeated below, it'd be good to put them into a function that has a comment on it explaining the purpose of these lines.

Reply:
Added a print statement.

Line 54 - Opens a shell terminal and 55 removes any existing marklogic.